### ISSUE TITLE
Document interface

### ISSUE THREAD

**Issue by moustachio-belvedere** [JuliaRheology/RHEOS.jl#99]
As Ale mentioned in RHEOSafm issue https://github.com/JuliaRheology/RHEOSafm.jl/issues/2, the RHEOS interface for contact models needs to be documented (and maybe tests added?)

I think once this is done, along with issue https://github.com/JuliaRheology/RHEOS.jl/issues/93 we should probably consider releasing version 0.9.4?

**Comment by akabla** [comment]
Yes, a new release would be good so that the new doc is made available soon.

**Comment by akabla** [comment]
@ab2425 
While working on the doc for this section, I realise that the AFM interface is not named properly. I think we should call it "Hertz" as it is more about the Hertz contact model, rather than about the instrument used. If we don't change it soon, we'll get stuck once we add more contact models later. What do you think?

Ale, I think this would break the link with RHEOSafm, so we need a synchronize update if we go this way.

**Comment by akabla** [comment]
Good progress here, tests added, importcsv improved to match the usage of the new version #115 
Documentation needs more examples, so moved to milestone 0.9.5.
